src/components/input/Input.tsx

```diff
@@ -25,59 +25,67 @@ interface InputProps
  * @param {boolean} [props.password] - 비밀번호 입력 필드로 설정합니다. `true`일 경우 비밀번호 숨기기/보기 토글 버튼이 활성화됩니다.
  */
 
-const Input = ({
-  type,
-  size,
-  error,
-  helperText,
-  disabled,
-  password,
-  ...rest
-}: InputProps) => {
-  const [passwordOff, setPasswordOff] = useState(true);
+const Input = forwardRef(
+  (
+    {
+      type,
+      size = 'large',
+      error,
+      helperText,
+      disabled,
+      password,
+      ...rest
+    }: InputProps,
+    ref: React.LegacyRef<HTMLInputElement>,
+  ) => {
+    const [passwordOff, setPasswordOff] = useState(true);
 
-  const handlePasswordState = () => {
-    setPasswordOff(!passwordOff);
-  };
+    const handlePasswordState = () => {
+      setPasswordOff(!passwordOff);
+    };
```

Comment:
```suggestion
    const handlePasswordState = () => {
      setPasswordOff(prev => !prev);
    };
```

함수형 업데이트로 state를 변경하면 최신의 상태값을 보장하면서 값을 업데이트 할 수 있어요!

[함수형 업데이트를 하는 이유.1](https://dev-astra.tistory.com/577)
[함수형 업데이트를 하는 이유.2](https://stackoverflow.com/questions/57828368/why-is-react-usestate-with-functional-update-form-is-needed)

Reply:
오오오오! 바로 반영하겟슴당 감사해여